applications/crossbar/src/modules_v1/cb_phone_numbers_v1.erl

```diff
@@ -418,8 +418,8 @@ get_find_numbers_req(Context) ->
 get_numbers(QueryString) ->
     PrefixQuery = kz_json:get_ne_value(?PREFIX, QueryString),
     Country = kz_json:get_ne_value(?COUNTRY, QueryString, ?DEFAULT_COUNTRY),
-    CountryPrefix = knm_util:prefix_for_country(Country),
-    Prefix = <<CountryPrefix/binary, PrefixQuery/binary>>,     
+    CountryPrefix = knm_iso3166a2_itu:to_itu(Country),
+    Prefix = <<CountryPrefix/binary, PrefixQuery/binary>>,
```

Comment:
i would prefer to leave `knm_util:prefix_for_country` and change its implementation to call ` knm_iso3166a2_itu:to_itu`
if/when we change to libpn we only have to change `knm_util:prefix_for_country`